SQL/Archive/18.0/20161115_media_unique_filename.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM media USING media, media m1 
+  WHERE media.date_uploaded < m1.date_uploaded AND media.file_name = m1.file_name;
```

Comment:
This will delete all records from media having a date_uploaded lower than any other date_uploaded for each file_name; Thus keep the latest record.